server/src/config.rs

```diff
@@ -46,6 +46,12 @@ pub struct StaticRouteConfig {
     pub topology: StaticTopologyConfig,
 }
 
+#[derive(Debug, Clone, Deserialize)]
+pub struct Node {
+    pub addr: String,
+    pub port: u16,
+}
```

Comment:
This is the same with `EndPoint`, do we have any way to reduce different naming on one thing?

Reply:
`Endpoint` is actually defined in `ceresdbproto` so we can't use it as some item of the configuration.